models/azure/storage_container_props.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2022 Specter Ops, Inc.
+//
+// This file is part of AzureHound.
+//
+// AzureHound is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// AzureHound is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+package azure
+
+type StorageContainerProperties struct {
```

Comment:
Ensure all properties are included and made public. Refer to: https://docs.microsoft.com/en-us/rest/api/storagerp/blob-containers/list?tabs=HTTP#listcontaineritem

Reply:
Got all properties(push to follow), however for the properties.metadata, it is documented as an object. I've tried setting some values in the interface, but the API does not return those in any format to see what I have to work with. Will leave it commented for the moment, unless I can find some examples.

Reply:
Defined as `interface{}`